pkg/backends/zenoss/tags_test.go

```diff
@@ -0,0 +1,126 @@
+package zenoss
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/atlassian/gostatsd"
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetTags(t *testing.T) {
+
+	t.Run("comparing values", func(t *testing.T) {
+		server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		}))
+		defer server.Close()
+
+		client, err := NewClient(server.URL, "apiKey123", false, false, 2, 2, 2, []string{"test:", "go"}, []string{"go", "lang"}, []string{"go", "stats"}, []string{"test:test_tag", "go"}, 1, float64(0.2), []string{"a", "b"}, 1*time.Second, gostatsd.TimerSubtypes{})
+		request1 := client.getTags([]string{"go:", "go"})
+		request2 := client.getTags([]string{"stats:", "abc"})
+		request3 := client.getTags([]string{"failed", "request"})
```

Comment:
It looks like you cover some of the variants in TestParseTag, bug still, i think these 3 test requests could be a bit more specific.

Maybe one with a tag name and value, one with a tag name ('true' value) and an invalid tag name, and one with nothing?

Reply:
added 4 specific requests: one with a tag name and value, one with a tag name ('true' value) and an invalid tag name, and one with nothing